tests/Offer/BulkLabelCommandHandlerTest.php

```diff
@@ -149,15 +149,13 @@ public function it_logs_an_error_when_an_error_occurred_and_continues_labelling(
         string $message
     ): void {
         // One label action should fail.
-        $this->commandBus->expects($this->at(0))
-            ->method('dispatch')
-            ->with(new AddLabel($this->offerIdentifiers[1]->getId(), $this->label))
-            ->willThrowException($exception);
-
         // Make sure the other offer is still labelled.
-        $this->commandBus->expects($this->at(1))
-            ->method('dispatch')
-            ->with(new AddLabel($this->offerIdentifiers[2]->getId(), $this->label));
+        $this->commandBus->method('dispatch')->will(
```

Comment:
Is it intentional to no longer test the parameters?

Reply:
The trouble, as far as I could find, is that on `withConsecutive()` or `willReturnMap()` you cannot throw an exception the first time it's run and than continue.

Reply:
I think you can add this
```
            ->withConsecutive(
                [new AddLabel($this->offerIdentifiers[1]->getId(), $this->label),],
                [new AddLabel($this->offerIdentifiers[2]->getId(), $this->label),]
            )
```

Reply:
Thanks, that's the trick🪄